selinux/README.md

```diff
@@ -1,142 +1,159 @@
-# CSI WekaFS SELinux Support
+# Add SELinux support
```

Comment:
This section refers only to SELinux enabled deployments different from RedHat OpenShift Container Platform (OCP)
For OCP, all necessary configurations are performed automatically

Reply:
Added the above to the overview